website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -128,7 +128,5 @@ For `Deploy Dev Cumulus and ORCA Stack` stage, add the following variables. The
 - DB_HOST_ENDPOINT
 - EARTHDATA_CLIENT_ID
 - EARTHDATA_CLIENT_PASSWORD
-- DLQ_SUBSCRIPTION_EMAIL

```

Comment:
I would definitely add verbiage in here that the buckets and Dynamo tables need to be manually setup with the proper prefix prior to running the pipeline.

Reply:
yup already added in last commit